versions/versions.md

```diff
@@ -0,0 +1,18 @@
+# Versions
+
+This document is meant to be a convenient reference to assist developers in checking out the desired old version of `Unlock.sol` or `PublicLock.sol` for reference/debugging purposes.
+The commit hashes represent the final state of the given contract prior to being upgraded or, in the case of the Lock contract, having the version number bumped.
+
+Use the command `$ git checkout <commit_hash_here>`to view the unlock repository at this point in time.
+
+After the upgrade to `Unlock.sol` V1, we'll need to modify our test suite to use the artifacts from this directory when testing Unlock V0, so: `const Unlock = artifacts.require('../../versions/Unlock_V0.json')`
+
+Note: It's possible that the artifacts.require syntax won't work for this use case. If this is the case, we'll need to look at using truffle-contract instead: https://github.com/trufflesuite/truffle/tree/master/packages/truffle-contract
+
+## Unlock.sol
+
+- `Unlock.sol` (V0) commit: <add commit# prior to upgrading to UnlockV1.sol>
```

Comment:
Would it make sense to list the mainnet and testnet addresses here as well?

Reply:
Hmm. We already have the address for each network in our zos network files (`zos.rinkeby.json`, etc) for the current version. This would be the logical place to record that info for old versions if you have a use case in mind where we would need it? 
Our test suite deploys a new version each run, and as for rinkeby/mainnet, having the old version address would mean we could interact with it directly, but not through the proxy (without setting the implementation to the old version). 
Perhaps I'll add the address for V0 now just in case...we can always remove if not needed.

Reply:
Done.

Reply:
This is not important, I was just thinking that it may be convenient.  For example, knowing the mainnet address of the Unlock v0 implementation (i.e. the code not just the proxy address) would allow us to easily confirm the bytecode is correct (e.g. check etherscan).